virtual_rainforest/models/animals/dummy_animal_module.py

```diff
@@ -242,4 +245,27 @@ def birth(self) -> AnimalCohort:
             An AnimalCohort instance having appropriate offspring traits for the
                 location and functional type of the parent cohort.
         """
+
         return AnimalCohort(self.functional_group, self.mass, 0, self.position)
+
+    def disperse(self, model: AnimalModel) -> None:
+        """The function to move a cohort between grid squares.
+
+        This is a simple implementation of what will need to be a much more complex
+        function. Right now it moves a cohort into a random neighboring grid, including
+        the current grid. Later functionality will need to involve bodysize, locomotion,
+        and other factors.
+
+        """
+        model.data.grid.set_neighbours(distance=sqrt(model.data.grid.cell_area))
```

Comment:
It may end up being useful to construct a neighbour matrix and store it as an animal cohort attribute as the experience of space is a function of the functional-type and body-size. In it's current form, I'm happy to store it in AnimalModel.__init__.